CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 # History
 
+## 3.4.0
```

Comment:
```suggestion
## 4.0.0
```

I think this is kinda breaking enough change for a major version bump, don't you think?

Reply:
Agree, but we should bump the version in gradle files as well.

Reply:
Yep, I just didn't comment on every other line with a version. :-)